index.html

```diff
@@ -12,12 +12,25 @@
       <div id="first">
         <div class="circle"></div>
       </div>
-      <div id="second"></div>
-      <div id="third"></div>
+
+      <div id="second">
+        <img
```

Comment:
try to style the image to be with original ratio proportions and make it centered with using of flex (hint, play around with flex positionin, align-items:center, justify-items:center etc...  also google it)

Reply:
Uptaded and image is now centered